web/packages/extension/global.d.ts

```diff
@@ -0,0 +1,8 @@
+declare global {
+    /**
+     * https://webpack.js.org/guides/public-path/#on-the-fly
+     */
+    let __webpack_public_path__: string;
```

Comment:
We used to have many more references to `__webpack_public_path__` in the codebase, but several were removed by #5017. I suppose we likely can't help adding this, but I though it worth noting that.